src/passes/stringify-walker.h

```diff
@@ -0,0 +1,68 @@
+#ifndef wasm_passes_stringify_walker_h
+#define wasm_passes_stringify_walker_h
+
+#include "ir/module-utils.h"
+#include "wasm-traversal.h"
+#include <queue>
+
+namespace wasm {
+
+/*
+ * This walker performs a shallow visit of control-flow (try, if, block, loop)
+ * expressions and their simple expression siblings before then visiting the
+ * children of each control-flow expression in postorder. As a result, this
```

Comment:
Perhaps a clearer version of this first sentence might be "This walker does a normal postorder traversal except that it defers traversing the contents of control flow structures it encounters."

Reply:
I prefer my wording because it better matches the implementation, which uses a queue to defer visiting the children.

Reply:
done

Reply:
I do still think it would be useful to say that this effectively un-nests the control flow, since I think that is a good higher-level intuitive explanation.

Reply:
done